allennlp/models/language_model.py

```diff
@@ -300,8 +302,9 @@ def forward(self,  # type: ignore
                     average_loss = forward_loss / num_targets.float()
             else:
                 average_loss = torch.tensor(0.0).to(forward_targets.device)  # pylint: disable=not-callable
-            # this is stored to compute perplexity if needed
+            # TODO: _last_average_loss buffer is no longer used.
```

Comment:
@brendan-ai2 , want to confirm that we're ok with just deleting `self._last_average_loss`?

Reply:
Removing this causes an error when restoring from a serialized model that was saved with this buffer. There is a test that fails when this is removed.

Reply:
Yeah, it's a bit annoying---we either have to retrain a new fixture or go into the existing fixture and manually remove the `_last_average_loss` key in the state dictionary. (1) might be easier, and we probably already have a process to do it... @brendan-ai2 might know more?

Reply:
The fact that the test with the fixture is breaking means that all serialized models would break if we remove it. This would be bad. Let's avoid that by just keeping the field around with a note that it's unused and exists for backwards compatibility.

Reply:
Sounds good. I added a comment above that buffer in `__init__` and removed the update from `forward`.